certified-connectors/PartnerLinq/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "username",
+          "description": "The username for this api",
+          "tooltip": "Provide the username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "password",
+          "description": "The password for this api",
+          "tooltip": "Provide the password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#e8e8e8",
+    "capabilities": [],
+    "publisher": "Visionet Systems Inc.",
+	"stackOwner":"Visionet Systems Inc."
```

Comment:
@FarhanJamilVSI all files should be properly formatted. Please fix the indentation for all files.